goupnp.go

```diff
@@ -72,19 +73,19 @@ type MaybeRootDevice struct {
 	Err error
 }
 
-// DiscoverDevices attempts to find targets of the given type. This is
+// DiscoverDevicesCtx attempts to find targets of the given type. This is
 // typically the entry-point for this package. searchTarget is typically a URN
 // in the form "urn:schemas-upnp-org:device:..." or
 // "urn:schemas-upnp-org:service:...". A single error is returned for errors
 // while attempting to send the query. An error or RootDevice is returned for
 // each discovered RootDevice.
-func DiscoverDevices(searchTarget string) ([]MaybeRootDevice, error) {
```

Comment:
Please keep the original *public* functions so that the original exists with the same signature in case existing code is using them directly. Obviously they can be trivially implemented in terms of the new functions.

Reply:
Yes. I did it on [Line 118](https://github.com/huin/goupnp/pull/54/files#diff-e85a736665f80e64a6a870ffde54d83abee17ea83321ee43291904512a093015R118)

Reply:
Ahh sorry - I didn't see that :)